### DISCUSSION TITLE
convert PDF error image loading

### DISCUSSION THREAD

**Discussion by khanht9x** [gotenberg/gotenberg#996] [Q&A]
I have an error in gotenberg 7.6 or lower when converting PDF from URL. When the image has not finished loading, gotenberg has converted it to PDF
Is there any way to fix this problem?
I have used the following parameter: ( NodeJS )

form.append('skipNetworkIdleEvent', 'true');
form.append("waitForExpression", "window.status === 'ready'");

I make 100 requests, 12 requests will have this situation

**Comment by gulien** [comment]
Hello @khanht9x,

Gotenberg `7.x` is no longer maintained so I cannot really help you here.

A few things to note:

* `form.append('skipNetworkIdleEvent', 'true');` => this form field may not be available in this version.
* `form.append("waitForExpression", "window.status === 'ready'");` => you have to add a script that set the status as "ready", it is not automatic.
